flashcards/backend/decks/tests/test_models_tag.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from ..models import Tag
+
+
+def test_tag_gui_representation():
+    tag = Tag(name="test_tag")
+
+    assert str(tag) == "test_tag"
+
+
+def test_tag_in_db(db, deck, flashcard):
+    tag = Tag(name="test_tag2", description="testDescription", deck=deck, flashcard=flashcard)
+    tag.save()
+
+    assert tag.name == "test_tag2"
+    assert tag.description == "testDescription"
+
+def test_tag_save_method_in_db(db, deck, flashcard):
```

Comment:
there is no check if it was saved to the database 😛

Reply:
The first test was not to be enrolled in the base